doc/getting-started.adoc

```diff
@@ -154,6 +154,41 @@ system configuration. It is recommender to copy and edit the configuration
 files from the `examples` directories if you are basing your configuration off
 of an example.
 
+== Usage with flakes ==
+
+Mobile NixOS is flake compatible, simply import the matching module:
+
+```nix
+# flake.nix
```

Comment:
Perhaps using a flake template would be convenient? The user could started with their own mobile-nixos configuration via:
`nix flake init -t github:NixOS/mobile-nixos`